src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -928,32 +928,89 @@ void EnIk_Update(Actor* thisx, PlayState* play2) {
-    -1, -1, -1, -1, -1, -1, 3,  5,  4,  6,  1,  2,  -1, -1, 0,  0,
+static s8 sLimbToArmorBodyParts[IRON_KNUCKLE_LIMB_MAX] = {
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_NONE
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_ROOT
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_TASSET_CENTER
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_TASSET_RIGHT
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_RIGHT_LEG
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_RIGHT_FOOT
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_TASSET_LEFT
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_LEFT_LEG
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_LEFT_FOOT
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_UPPER_BODY_ROOT
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_UNK_ROOT
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_HEAD_ROOT
+    IRON_KNUCKLE_ARMOR_BODYPART_HELMET,               // IRON_KNUCKLE_LIMB_HELMET_ARMOR
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_HEAD
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_RIGHT_UPPER_ARM
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_RIGHT_FOREARM
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_AXE_ROOT
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_AXE
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_RIGHT_HAND
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_LEFT_UPPER_ARM
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_LEFT_FOREARM
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_LEFT_HAND
+    IRON_KNUCKLE_ARMOR_BODYPART_UPPER_LEFT_PAULDRON,  // IRON_KNUCKLE_LIMB_UPPER_LEFT_PAULDRON
+    IRON_KNUCKLE_ARMOR_BODYPART_LOWER_LEFT_PAULDRON,  // IRON_KNUCKLE_LIMB_LOWER_LEFT_PAULDRON
+    IRON_KNUCKLE_ARMOR_BODYPART_UPPER_RIGHT_PAULDRON, // IRON_KNUCKLE_LIMB_UPPER_RIGHT_PAULDRON
+    IRON_KNUCKLE_ARMOR_BODYPART_LOWER_RIGHT_PAULDRON, // IRON_KNUCKLE_LIMB_LOWER_RIGHT_PAULDRON
+    IRON_KNUCKLE_ARMOR_BODYPART_CHEST_FRONT,          // IRON_KNUCKLE_LIMB_CHEST_ARMOR_FRONT
+    IRON_KNUCKLE_ARMOR_BODYPART_CHEST_BACK,           // IRON_KNUCKLE_LIMB_CHEST_ARMOR_BACK
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_TORSO
+    BODYPART_NONE,                                    // IRON_KNUCKLE_LIMB_WAIST
 };
 
 s32 EnIk_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
     EnIk* this = THIS;
 
     if (this->drawArmorFlags) {
```

Comment:
```suggestion
    if (this->drawArmorFlags != 0) {
```
No you, but seems similar to the other style below.